### PR TITLE
capacity management dashboard

### PR DESCRIPTION
**Pull request by raffaelespazzoli**

### CODE REVIEW

File: capacity-dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+```shell
+git clone https://github.com/redhat-cop/openshift-toolkit
+cd openshift-toolkit/custom-grafana
+ansible-galaxy install -r requirements.yml -p galaxy
+ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml \
+  -e include_tags=""
+```
+
+Open the custom Grafana and import the capacity planning dashboard: [capacity-planning.json](./capacity-planning.json)
```
Comment (etsauer) [inline capacity-dashboard/README.md]: @raffaelespazzoli take a look at how we are deploying each dashboard in the Custom Dashboards applier. We should be able to just add this dashboard in a similar way via configmap.

Comment (etsauer) [reply capacity-dashboard/README.md]: for example: https://github.com/redhat-cop/openshift-toolkit/blob/master/custom-dashboards/.applier/group_vars/all.yml#L63-L76

File: capacity-dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+### Interpreting the quota vs allocatable ratio panel
+
+This ration shows what we have promised to the team we can give them in terms of resources versus what we have. This is a slow moving ratio because it changes only when new quotas are granted, which should more or less coincide with new teams being onboarded to the cluster. This metric can be used to make long term (and therefore non-precise) forecast of what is going to be needed. This metrics is best suited when our node provisioning lifecycle is very long: weeks or months. This likely happens when the nodes are physical machines.
+With this ratio is possible and ok to go above 100%. In that situation the cluster is over committed. Over committing the cluster is generally fine if we can safely assume that not all the projects will use all of their quotas at the same time. Each organization will have to find its own sweet spot in terms of over committing the cluster.
+
+## Installation
+
+Install the openshift-state-metrics additional metrics here as follows
```
Comment (etsauer) [inline capacity-dashboard/README.md]: @raffaelespazzoli FYI, openshift state metrics is already installed by the custom dashboard project.

File: capacity-dashboard/README.md (hunk 1 above)
Comment (raffaelespazzoli) [reply capacity-dashboard/README.md]: should be fine now

File: capacity-dashboard/README.md (hunk 2 above)
Comment (raffaelespazzoli) [reply capacity-dashboard/README.md]: are we using the latest with that approach? I saw that Shah was not using the latest.

Comment (raffaelespazzoli) [reply capacity-dashboard/README.md]: the latest push should address the concern.
If there was a way for applier to get the manifests directly from the openshift state metrics repo, I would recommend doing that.

**Pull request merged**